tests/std/tests/P0980R1_constexpr_strings/test.cpp

```diff
@@ -2383,6 +2383,19 @@ constexpr void test_all() {
     static_assert(test_allocator_awareness<CharType>());
 }
 
+#if _HAS_CXX23
+void test_gh_2524() // COMPILE-ONLY
+{

```

Comment:
No change requested (mega-nitpick, not worth resetting testing): This brace should be attached to the function. (clang-format would ordinarily fix this, but it's inhibited from doing so by the comment.)